cmd/ctrl.go

```diff
@@ -45,8 +49,21 @@ type capture struct {
 
 // cfMeta holds changefeed info and changefeed status
 type cfMeta struct {
-	Info   *model.ChangeFeedInfo   `json:"info"`
-	Status *model.ChangeFeedStatus `json:"status"`
+	Info       *model.ChangeFeedInfo   `json:"info"`
+	Status     *model.ChangeFeedStatus `json:"status"`
+	Count      uint64                  `json:"count"`
+	TaskStatus []captureTaskStatus     `json:"task-status"`
+}
+
+type captureTaskStatus struct {
+	CaptureID  string            `json:"capture-id"`
+	TaskStatus *model.TaskStatus `json:"status"`
+}
+
+type profileStatus struct {
+	OPS   uint64 `json:"ops"`
+	Gap   int64  `json:"gap"`
```

Comment:
What about adding another status, which tells user the replication gap between `current tso` and `checkpoint ts`. It may be a more native metric about replication status behind upstream